packages/dashboard/src/components/Card/Card.types.ts

```diff
@@ -11,22 +11,27 @@ export enum CardSize {
   /**
    * Option for selecting small card
    */
-  small,
+  small = 'small',
```

Comment:
>small = 'small', [](start = 2, length = 16)

why change this is to string? Have you seen if this breaks the cards in fabric example?

Reply:
a few things to clarify:
- this is a string enum in typescript which is quite useful  [https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-4.html](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-4.html)
- this is an effort to merge Size and CardSize and the size enum in DashboardGridLayout.types.ts is already a string enum
- the cards are working fine as far as I can tell.

---
In reply to: [211799641](https://github.com/OfficeDev/office-ui-fabric-react/pull/5899#discussion_r211799641) [](ancestors = 211799641)